src/pages/Resume/EmploySearchDetailPage.test.tsx

```
@@ -0,0 +1,246 @@
+    it('useGetApplicantResume이 올바른 매개변수로 호출되어야 한다', () => {
+      render(<EmploySearchDetailPage />, { wrapper: createWrapper() });
+
+      expect(mockUseGetApplicantResume).toHaveBeenCalledWith(1, true);
+    });
+
+    it('API 응답이 없을 때 기본값이 설정되어야 한다', () => {
+      mockUseGetApplicantResume.mockReturnValue({
+        data: null,
+        isLoading: false,
+        error: null,
+      } as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)